src/main/resources/application.yml

```diff
@@ -65,6 +65,10 @@ nakadi:
     send.timeoutMs: 5000
     batch.size: 5242880
     max.request.size: 2098152
+    buffer.memory: 83886080 # May be defined as X * Y  * Z, where:
```

Comment:
Is this memory per producer? Do we have a single producer per instance (sorry a cannot remember anymore)?

Reply:
We have as many producers as active storages.
There is also a case when producers are dying, and it may be temporary more than 1 producer per active storage. So it should be fine.

Reply:
Btw, default kafka value is 32 Mb. I changed to 80 Mb

Reply:
Ok 👍  sounds good